db/seeds.rb

```diff
@@ -133,11 +134,20 @@ def make_purchases
   end
 end
 
+def make_subcategories
+  names = ['Science Fiction', 'Historical Fiction', 'Fantasy']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.